fetch_release.py

```diff
@@ -134,7 +134,7 @@ def get_micromamba(version, use_default_version):
 
         # compute the sha256
         sha256 = hashlib.sha256()
-        with open(outdir / f"micromamba-{dplat}", "rb") as f:
+        with open(outdir / f"micromamba-{dplat}{ext}", "rb") as f:
```

Comment:
Thanks for tackling this.
There are actually 2 checksums to compute and write (one for the archive and another for the executable) and they need to be written in the corresponding `.sha256` files.
So we would need to add another sha256 for either the executable (if you change this line) or leave this as is and add the one for the archive.